### DISCUSSION TITLE
choice of pair_style in LAMMPS

### DISCUSSION THREAD

**Discussion by rvillarparajo** [CederGroupHub/chgnet#129] [Announcements]
Hi!

It might be too much of a basic question, but I am trying to run MDs with CHGnet in LAMMPS but I am unable to find clarificatino on what chgnet/d3 does differently than chgnet. Could you help me with that?

Many thanks

**Comment by chtchelkatchev** [comment]
Look at https://github.com/advancesoftcorp/lammps/blob/based-on-lammps_2Jun2022/potentials/CHGNET/chgnet_driver.py
It seems that chgnet/d3 adds DFT-d3 corrections to the interatom potential. See also https://github.com/dftd3

**Reply by wugaxp** [reply]
My experience is that chgnet/d3 is much slower than the chgnet pair_style for modest size system (~2000 atoms), perhaps due to the exchange of data between gpu and cpu?

**Comment by bowen-bd** [comment]
DFT-D3 adds the vdw correction that is missing in DFT, thanks for @chtchelkatchev 's answer
